<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SHEETAL-EVA" failures="1" tests="1" name="Regression Test" time="9.12" errors="0" timestamp="2022-11-29T14:00:39 IST">
  <testcase classname="ai.dalalstreet.testLayer.TestLoginPage" name="verifyLoginpage" time="3.041">
    <failure type="java.lang.AssertionError" message="expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://apps.dalalstreet.ai/dashboard]">
      <![CDATA[java.lang.AssertionError: expected [https://www.apps.dalalstreet.ai/dashboard] but found [https://apps.dalalstreet.ai/dashboard]
at ai.dalalstreet.testLayer.TestLoginPage.verifyLoginpage(TestLoginPage.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginpage -->
</testsuite> <!-- Regression Test -->
